src/code/z_parameter.c

```diff
@@ -3128,10 +3127,8 @@ void Inventory_UpdateDeitySwordEquip(PlayState* play) {
         interfaceCtx->unk_21C = 0;
         interfaceCtx->bButtonDoAction = 0;
 
-        // Is simply checking if (gSaveContext.save.playerForm == PLAYER_FORM_FIERCE_DEITY)
-        if ((((gSaveContext.save.playerForm > 0) && (gSaveContext.save.playerForm < 4))
-                 ? 1
-                 : gSaveContext.save.playerForm >> 1) == 0) {
+        // Is simply checking if (GET_PLAYER_FORM == PLAYER_FORM_FIERCE_DEITY)
+        if ((((GET_PLAYER_FORM > 0) && (GET_PLAYER_FORM < 4)) ? 1 : GET_PLAYER_FORM >> 1) == 0) {
```

Comment:
I'm actually conflicted if we should use the PLAYER enum here. It technically makes sense, but the the way it does the check is just weird.

Reply:
I'm leaning towards not because `>> 1` actually depends on the value of the enum, instead of just being a label but idk, I'd be fine if it were put in.